src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -70,12 +70,12 @@ namespace OpenMS
   {
 public:
 
-    inline static ModificationsDB * getInstance()
+    inline static ModificationsDB * getInstance(OpenMS::String unimod_file = "CHEMISTRY/unimod.xml", OpenMS::String psimod_file = "CHEMISTRY/PSI-MOD.obo")
     {
       static ModificationsDB * db_ = 0;
       if (db_ == 0)
       {
-        db_ = new ModificationsDB;
+        db_ = new ModificationsDB(unimod_file, psimod_file);
```

Comment:
this kind of solves the problem, basically whoever calls `getInstance` determines what the unimod and psimod file will be like. 

As long as we do not have any calls to `getInstance` during the startup of OpenMS and the TOPPTools, that should work, right? It is not a very elegant solution but I think it should work. any comments?